src/main/kotlin/com/frcteam3636/frc2025/subsystems/drivetrain/DrivetrainIO.kt

```diff
@@ -29,12 +31,13 @@ abstract class DrivetrainIO {
         modules.forEach(SwerveModule::periodic)
 
         inputs.gyroRotation = gyro.rotation
+        inputs.gyroVelocity = gyro.velocity
+        inputs.gyroConnected = gyro.connected
         inputs.measuredStates = modules.map { it.state }
         inputs.measuredPositions = modules.map { it.position }
-        inputs.gyroConnected = gyro.connected
     }
 
-    fun setGyro(rotation: Rotation3d) {
+    fun setGyro(rotation: Rotation2d) {
         gyro.rotation = rotation
```

Comment:
`rotation` could be checked by an assert call.

Reply:
How so?

Reply:
Just check if some crazy angle gets passed in. (e.g an angle > 360 or < 0